packages/mobile/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run pre-commit
```

Comment:
Where is this coming from? I thought mookme was handling all the git hooks

Reply:
yup good catch, .gitignore not catching